### PR TITLE
add support laravel v6

### LINKED ISSUE #114: Laravel 6 LTS Support
Hello, 

Today Laravel 6 LTS was released, will it be supported/updated for this version? 

Thank you

### PR DESCRIPTION
**Pull request by marcelogarbin**
fix #114

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,12 +11,12 @@
     ],
     "require": {
         "php": "^5.6.4 || ^7.0",
-        "illuminate/support": "^5.0",
+        "illuminate/support": "^5.0|6.0.*",
```
Comment (it-can) [inline composer.json]: use ```^6.0```  instead of ```6.0.*```

File: composer.json [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
         "guzzlehttp/guzzle": "^6.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.1|^7.0",
-        "illuminate/view": "^5.0"
+        "phpunit/phpunit": "^8.0",
+        "illuminate/view": "^5.0|6.0.*"
```
Comment (it-can) [inline composer.json]: use ```^6.0```  instead of ```6.0.*```

**Comment by it-can** [comment]
@marcelogarbin can you change ```6.0.*``` to ```^6.0``` for ```illuminate/support``` and ```illuminate/view``` ?

**Comment by marcelogarbin** [comment]
@it-can Yes, I did that. thanks for reviewing

**Comment by it-can** [comment]
@albertcht can you merge this?

**Comment by marcelogarbin** [comment]
Unfortunately I am not an administrator. I'm waiting for the @albertcht  check this out

**Comment by albertcht** [comment]
Hi @marcelogarbin ,

Thanks for your pull request.

**Comment by MichaelBelgium** [comment]
@albertcht Please add the support to the multi-forms branch also 😄

**Pull request merged**